src/sentry/utils/snuba.py

```diff
@@ -172,6 +171,39 @@ def log_snuba_info(content):
     "tags.value": "tags.value",
 }
 
+SPAN_EAP_COLUMN_MAP = {
+    "id": "span_id",
+    "organization.id": "organization_id",
+    "project": "project_id",
+    "project.id": "project_id",
+    "project_id": "project_id",
+    "span.action": "attr_str[action]",
+    # For some reason the decision was made to store description as name? its called description everywhere else though
+    "span.description": "name",
+    "description": "name",
+    # message also maps to span description but gets special handling
+    # to support wild card searching by default
+    "message": "name",
+    "span.domain": "domain",
+    "span.group": "group",
+    "span.op": "attr_str[op]",
+    "span.category": "attr_str[category]",
+    "span.self_time": "exclusive_time_ms",
+    "span.status": "attr_str[status]",
+    "timestamp": "start_timestamp",
+    "trace": "trace_id",
+    "transaction": "segment_name",
+    "transaction.id": "segment_id",
+    "is_transaction": "is_segment",
+    "segment.id": "segment_id",
+    # We should be able to delete this and just use transaction
+    "origin.transaction": "segment_name",
+    # These are guesses:
+    "transaction.op": "attr_str[transaction_op]",
```

Comment:
attr_str[transaction.op]

Reply:
full list is this:

```
pub fn sentry_tag_key(&self) -> &str {
        match self {
            SpanTagKey::Release => "release",
            SpanTagKey::User => "user",
            SpanTagKey::UserID => "user.id",
            SpanTagKey::UserUsername => "user.username",
            SpanTagKey::UserEmail => "user.email",
            SpanTagKey::Environment => "environment",
            SpanTagKey::Transaction => "transaction",
            SpanTagKey::TransactionMethod => "transaction.method",
            SpanTagKey::TransactionOp => "transaction.op",
            SpanTagKey::Mobile => "mobile",
            SpanTagKey::DeviceClass => "device.class",
            SpanTagKey::BrowserName => "browser.name",
            SpanTagKey::SdkName => "sdk.name",
            SpanTagKey::SdkVersion => "sdk.version",
            SpanTagKey::Platform => "platform",

            SpanTagKey::Action => "action",
            SpanTagKey::AIPipelineGroup => "ai_pipeline_group",
            SpanTagKey::Category => "category",
            SpanTagKey::Description => "description",
            SpanTagKey::Domain => "domain",
            SpanTagKey::RawDomain => "raw_domain",
            SpanTagKey::Group => "group",
            SpanTagKey::HttpDecodedResponseContentLength => "http.decoded_response_content_length",
            SpanTagKey::HttpResponseContentLength => "http.response_content_length",
            SpanTagKey::HttpResponseTransferSize => "http.response_transfer_size",
            SpanTagKey::ResourceRenderBlockingStatus => "resource.render_blocking_status",
            SpanTagKey::SpanOp => "op",
            SpanTagKey::SpanStatus => "status",
            SpanTagKey::StatusCode => "status_code",
            SpanTagKey::System => "system",
            SpanTagKey::TimeToFullDisplay => "ttfd",
            SpanTagKey::TimeToInitialDisplay => "ttid",
            SpanTagKey::FileExtension => "file_extension",
            SpanTagKey::MainThread => "main_thread",
            SpanTagKey::CacheHit => "cache.hit",
            SpanTagKey::CacheKey => "cache.key",
            SpanTagKey::OsName => "os.name",
            SpanTagKey::AppStartType => "app_start_type",
            SpanTagKey::ReplayId => "replay_id",
            SpanTagKey::TraceStatus => "trace.status",
            SpanTagKey::MessagingDestinationName => "messaging.destination.name",
            SpanTagKey::MessagingMessageId => "messaging.message.id",
            SpanTagKey::ThreadName => "thread.name",
            SpanTagKey::ThreadId => "thread.id",
            SpanTagKey::ProfilerId => "profiler_id",
        }
    }
```

Reply:
@colin-sentry can you link where you pulled that from so I can reference it in the comments

Reply:
Wait why does this list have `transaction` i thought its called segment in this new table? Did that finally get fixed and its called transaction now?

Reply:
https://github.com/getsentry/relay/blob/f2980a8bb45dbf69fc408d05615b022bb888518f/relay-event-normalization/src/normalize/span/tag_extraction.rs#L32